tests/acceptance/features/apiUserLDAPSharing/sharingLocalUserLdapUser.feature

```diff
@@ -0,0 +1,125 @@
+   When user "user0" uploads file with content "new file" to "PARENT (2)/new-file.txt" using the WebDAV API
+   Then the HTTP status code should be "403"
+   And as "local-user" file "/PARENT/new-file.txt" should not exist
+   When user "user0" uploads file with content "changed file" to "PARENT (2)/parent.txt" using the WebDAV API
+   Then the HTTP status code should be "403"
+   And the content of file "/PARENT/parent.txt" for user "local-user" should be:
+    """
+    ownCloud test text file parent
+    
+    """
+
+  Scenario: Share a folder from an LDAP user to a local group, delete the group
+    Given group "local-group" has been created in the database user backend
+    And user "local-user" has been added to group "local-group"
+    When user "user0" shares folder "/PARENT" with group "local-group" using the sharing API
+    Then the content of file "/PARENT (2)/parent.txt" for user "local-user" should be:
+    """
+    ownCloud test text file parent
+    
+    """
+    When the administrator deletes group "local-group" using the provisioning API
+    Then as "local-user" file "/PARENT (2)/parent.txt" should not exist
+
+  Scenario: Share a folder from a local user to an LDAP group, delete the group
+    When user "local-user" shares folder "/PARENT" with group "grp1" using the sharing API
+    Then the content of file "/PARENT (2)/parent.txt" for user "user1" should be:
+    """
+    ownCloud test text file parent
+    
+    """
+    And the content of file "/PARENT (2)/parent.txt" for user "user2" should be:
+    """
+    ownCloud test text file parent
+    
+    """
+    When the administrator deletes ldap group "grp1"
+    Then as "user1" file "/PARENT (2)/parent.txt" should not exist
+    And as "user2" file "/PARENT (2)/parent.txt" should not exist
+
+  Scenario: Share a folder from an LDAP user to a local group, take member out of the group
+    Given group "local-group" has been created in the database user backend
+    And user "local-user" has been added to group "local-group"
+    When user "user0" shares folder "/PARENT" with group "local-group" using the sharing API
+    Then the content of file "/PARENT (2)/parent.txt" for user "local-user" should be:
+    """
+    ownCloud test text file parent
+    
+    """
+    When the administrator removes user "local-user" from group "local-group" using the provisioning API
+    Then as "local-user" file "/PARENT (2)/parent.txt" should not exist
+
+  Scenario: Share a folder from a local user to an LDAP group, take member out of the group
+    When user "local-user" shares folder "/PARENT" with group "grp1" using the sharing API
+    Then the content of file "/PARENT (2)/parent.txt" for user "user1" should be:
+    """
+    ownCloud test text file parent
+    
+    """
+    And the content of file "/PARENT (2)/parent.txt" for user "user2" should be:
+    """
+    ownCloud test text file parent
+    
+    """
+    When the administrator removes user "user1" from ldap group "grp1"
```

Comment:
Can you do this?
I thought that LDAP group membership comes read-only from the LDAP server.

Reply:
yes, but this step does manipulate the LDAP server itself, it does not do anything with the oC server.

Reply:
:+1: